front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -188,24 +180,17 @@ const AdminProjectsList = memo(({ className }: Props) => {
                 </Box>
               </Tooltip>
             )}
-            <Tooltip
-              content={
-                <FormattedMessage {...messages.onlyAdminsCanCreateProjects} />
-              }
-              disabled={userCanCreateProject}
-            >
-              <Box>
-                <Button
-                  data-cy="e2e-new-project-button"
-                  linkTo={'/admin/projects/new'}
-                  icon="plus-circle"
-                  buttonStyle="admin-dark"
-                  disabled={!userCanCreateProject}
-                >
-                  <FormattedMessage {...messages.newProject} />
-                </Button>
-              </Box>
-            </Tooltip>
+
+            <Box>
+              <Button
+                data-cy="e2e-new-project-button"
+                linkTo={'/admin/projects/new'}
+                icon="plus-circle"
+                buttonStyle="admin-dark"
+              >
+                <FormattedMessage {...messages.newProject} />
+              </Button>
```

Comment:
Just to confirm, this means that any Admin, Folder Mod, or Project Mod can not create new projects then right?

Reply:
Yes, draft projects. Project moderators cannot publish them the first time without approval. Folder moderators and admin can publish projects at any time.